Sources/MapboxDirections/MBLane.swift

```diff
@@ -3,19 +3,30 @@ import Foundation
 /**
  A lane on the road approaching an intersection.
  */
+@objcMembers
 @objc(MBLane)
 public class Lane: NSObject, NSSecureCoding {
     /**
      The lane indications specifying the maneuvers that may be executed from the lane.
      */
+    #if SWIFT_PACKAGE
+    public let indications: LaneIndication
+    #else
     @objc public let indications: LaneIndication
+    #endif
     
     /**
      Initializes a new `Lane` using the given lane indications.
      */
+    #if SWIFT_PACKAGE
```

Comment:
I'm not entirely satisfied with this workaround, having to provide duplicated initializers.
What I would like to see is a pre-processor macro:
```
#if SWIFT_PACKAGE
#define OBJC_NO_SPM
#define OBJC_NO_SPM(x)
#else
#define OBJC_NO_SPM @objc
#define OBJC_NO_SPM(x) @objc(x)
#endif
```
but an Obj-C macro would not work with SPM and SPM doesn't support these macros.

`@objcMembers` solves roughly half of the issues, since it overrides the `@objc`-flag if it's not able to bridge the to Obj-C, but it only works for some methods and properties.